server/main.go

```diff
@@ -45,8 +45,14 @@ func main() {
 	ServicesApiService := openapi.NewServicesApiService(kuskClient)
 	ServicesApiController := openapi.NewServicesApiController(ServicesApiService)
 
-	StaticRouteApiService := openapi.NewCreateNewStaticRouteApiService(kuskClient)
-	StaticRouteApiController := openapi.NewCreateNewStaticRouteApiController(StaticRouteApiService)
+	StaticCreateRouteApiService := openapi.NewCreateNewStaticRouteApiService(kuskClient)
+	StaticCreateRouteApiController := openapi.NewCreateNewStaticRouteApiController(StaticCreateRouteApiService)
+
+	StaticRouteApiService := openapi.NewStaticRouteApiService(kuskClient)
+	StaticRouteApiController := openapi.NewStaticRouteApiController(StaticRouteApiService)
```

Comment:
@mbana can you recall when doing StaticCreateRouteApiController were there any other controllers generated by the code generator?

Somehow it created CreateRoute, Routes and Route controllers